### ISSUE TITLE
`logging-too-many-args (PLE1205)` isn't detected when initialized logger is imported from another module

### ISSUE THREAD

**Issue by osauldmy** [astral-sh/ruff#15314]
When `logger` isn't initialized in the module, but imported from another module like a singleton (which may be an antipattern anyway), `PLE1205` isn't detected.

Minimal reproducible example:

```python
# direct.py
import logging

logger = logging.getLogger(__name__)

logger.info("Foo", "bar")
```

```python
# foo/logger.py
import logging

logger = logging.getLogger(__name__)
```

```python
# imported.py
from foo.logger import logger

logger.info("Foo", "bar")
```

```shell
$ ruff check --output-format=concise --select PLE
direct.py:5:1: PLE1205 Too many arguments for `logging` format string
Found 1 error.
```

Expected behaviour:

```shell
$ ruff check --output-format=concise --select PLE
direct.py:5:1: PLE1205 Too many arguments for `logging` format string
imported.py:3:1: PLE1205 Too many arguments for `logging` format string
Found 2 errors.
```

**Comment by MichaReiser** [comment]
This is a known limitation of Ruff today because it doesn't support multifile analysis. You can work around it by adding `foo.logger.logger` to [`logger-objects`](https://docs.astral.sh/ruff/settings/#lint_logger-objects)

**Comment by osauldmy** [comment]
I completely forgot about `logger-objects` option. Thanks!
